File: front-end/src/app/service-models/app/v1/app-users/application-user-address-s-m.ts
import { CoinManagementServiceModelBase } from '../../base/coin-management-service-model-base';

export class ApplicationUserAddressSM extends CoinManagementServiceModelBase<number> {
    country!: string;
    state!: string;
    city!: string;
    address1!: string;
    address2!: string;
    pinCode!: string;
    applicationUserId!: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/AppUsers/ApplicationUserAddressSM.cs
namespace Siffrum.Web.Payroll.ServiceModels.v1.AppUsers
{
    public class ApplicationUserAddressSM : Base.SiffrumPayrollServiceModelBase<int>
    {
        public string Country { get; set; }

        public string State { get; set; }

        public string City { get; set; }

        public string Address1 { get; set; }

        public string Address2 { get; set; }

        public string PinCode { get; set; }

        public int ApplicationUserId { get; set; }
    }
}
